context/app/static/js/pages/CellsAPIDemo/DatasetsSelectedByExpression.jsx

```diff
@@ -38,6 +39,45 @@ function DatasetsSelectedByExpression(props) {
       setMessage(e.message);
     }
   }
+  const query = useMemo(() => {
+    return {
+      query: {
+        bool: {
+          must_not: {
+            exists: {
+              field: 'next_revision_uuid',
+            },
+          },
+        },
+      },
+      post_filter: {
+        bool: {
+          must: [
+            {
+              term: {
+                'entity_type.keyword': 'Dataset',
+              },
+            },
+            {
+              terms: {
+                uuid: results.map((result) => result.uuid),
+              },
+            },
+          ],
+        },
+      },
+      _source: [
+        'uuid',
+        'hubmap_id',
+        'mapped_data_types',
+        'origin_sample.mapped_organ',
+        'donor.mapped_metadata',
+        'last_modified_timestamp',
```

Comment:
```suggestion
        'mapped_last_modified_timestamp',
```

Reply:
I think the design specifies 'yyyy-MM-dd` which doesn't match the mapped timestamp, I'd prefer to format using the timestamp rather than the string.